agent-lib/src/main/java/org/terracotta/angela/agent/client/RemoteClientManager.java

```diff
@@ -108,7 +108,9 @@ public int spawnClient(InstanceId instanceId, TerracottaCommandLineEnvironment t
       cmdLine.add("-classpath");
       cmdLine.add(buildClasspath());
 
-      cmdLine.add("-Dangela.port=" + portAllocator.getNewRandomFreePorts(2).getBasePort());
+      try (PortAllocator.PortAllocation portAllocation = portAllocator.reserve(2) ) {
+        cmdLine.add("-Dangela.port=" + portAllocation.next());
```

Comment:
@saurabhagas : the PR is wrong...

See my comment: https://github.com/Terracotta-OSS/terracotta-utilities/pull/14#issuecomment-623664922

The new port manager is not reserving consecutive port numbers but random ones...

Ignite needs 2 ports.

So now, we need to do something lile:

```
cmdLine.add("-Dignite.discovery.port=" + portAllocation.next());
cmdLine.add("-Dignite.com.port=" + portAllocation.next());
```

And read these 2 ports from the agent